### PR TITLE
feature: added mycart page

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added E-Commerce My Cart Page

Live Link: https://neog-e-comm-website.netlify.app/mycart/index.html

### CODE REVIEW

File: mycart/mycartstyles.css [hunk 1]
```diff
@@ -1,145 +1,43 @@
...
+.grid-container {
     display: grid;
     grid-template-columns: auto auto;
-}
\ No newline at end of file
+}
+.move-to-cart{
+    width: 20rem;
+    padding: 0.4rem;
+    margin-top: 0.90rem;
```
Comment (ranitasahaa1230) [inline mycart/mycartstyles.css:38]: You can use px for these values

**Review by ranitasahaa1230 — APPROVED** [review]
Rest, looks good to me

File: mycart/mycartstyles.css (hunk 1 above)
Comment (poojakelkar) [reply mycart/mycartstyles.css:38]: yeah we can add both Thanks For Review @ranitasahaa1230

**Review by chanchal-panpaliya — APPROVED** [review]
UI and code looks good to merge .

**Comment by poojakelkar** [comment]
Thanks @chanchal-panpaliya and @ranitasahaa1230  for PR

**Pull request merged**